### DISCUSSION TITLE
[Suggestion] Rotor Limit

### DISCUSSION THREAD

**Discussion by TheVikingWalrus** [tsunamayo/Starship-EVO#4002] [Suggestions]
I would like to see changes to the rotor limit as it does really affect large builds that would feature some things such as rotating rings and such. I have had a look at what other people have said about the rotor limit and have seen that large rotor items can really cause lag and such due to the amount of blocks being calculated for hitboxes or such, and I can totally understand the limitations for that. I think (although not as a priority as I am sure there is more important things to do, such as the voxel removal and such) is to try and see if that can be solved or if it cannot be solved, remove the limit, and instead replace it with warnings.

I feel that this is of some importance to make a request for it as space stations with rotating rings are such an icon for space, and allows for builds with such character, where you can look out a window and see space rotating around you. Once again, I understand the technical limitations being faced, but I feel like it shouldn't be left with the system as is indefinitely, rather be checked again in the future to see if such can be resolved.

**Comment by tsunamayo** [comment]
@TheVikingWalrus Hi, yes I want to work on that in the future. The new collision framework is much more powerful now. Thanks
